src/main/resources/lang/default.lang

```diff
@@ -2413,6 +2413,34 @@ item display transforms:
 	thirdperson_lefthand: third person left handed, third person left hand, left handed in third person
 	thirdperson_righthand: third person right handed, third person right hand, right handed in third person
 
+# -- Villager Types/Professions --
+villager types:
+	snow: snow
+	plains: plains
+	jungle: jungle
+	taiga: taiga
+	desert: desert
+	savanna: savanna
+	swamp: swamp
+
+villager professions:
+	# Suffix "profession" options added to help with EntityData conflicts
+	leatherworker: leatherworker, leatherworker profession
+	mason: mason, mason profession
+	fletcher: fletcher, fletcher profession
+	weaponsmith: weaponsmith, weaponsmith profession
+	toolsmith: toolsmith, toolsmith profession
+	librarian: librarian, librarian profession
+	shepherd: shepherd, shepherd profession
+	farmer: farmer, farmer profession
+	cleric: cleric, cleric profession
+	nitwit: nitwit, nitwit profession
+	cartographer: cartographer, cartographer profession
+	armorer: armorer, armorer profession
+	butcher: butcher, butcher profession
+	none: none, none profession, no profession, unemployed
```

Comment:
worried about none being an option and therefore ``if {_X} is none`` being valid syntax.
I would rather see this be just the latter two options.

Reply:
Good call, I will remove that.